portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -25,18 +25,24 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import java.time.format.DateTimeFormatter;
+import java.time.*; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/* Servlet that creates a comment*/
 @WebServlet("/new-comment")
 public class NewCommentServlet extends HttpServlet {
   
+  private static final DateTimeFormatter commentDateFormat = DateTimeFormatter.ofPattern("yyyy/MM/dd");
+  
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("commentText");
     long timestamp_millis = System.currentTimeMillis();
-
+    String currentDate = commentDateFormat.format(LocalDateTime.now());
+    
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("content", text);
+    commentEntity.setProperty("currentDate", currentDate);
     commentEntity.setProperty("timestamp_millis", timestamp_millis);
```

Comment:
Just a note, not required to change it, but this logic could be moved to the Comment class with a method like:

```
Comment comment = new Comment (...);
Comment.CopyTo(commentEntity);
```

CopyTo() would set all the properties on `commentEntity`.